src/gnssro/CMakeLists.txt

```diff
@@ -1,18 +1,23 @@
 # src/gnssro/CMakeLists.txt
 #
 # Build 1) gnssro ncepbufr to ioda NetCDF4 converter
-#       2) gnssro gsi ncdiag to ioda NetCDF4 converter
-
+#       2) gnssro ncepbufr to ioda2 NetCDF4 converter
+#       3) gnssro gsi ncdiag to ioda NetCDF4 converter
+         
 ecbuild_add_executable( TARGET  gnssro_bufr2ioda
                         SOURCES gnssro_bufr2ioda.f90
                         LIBS    bufr::bufr_4_DA
                                 NetCDF::NetCDF_Fortran)
 
+ecbuild_add_executable( TARGET  gnssro_bufr2ioda2
```

Comment:
Why is there a new executable?

If the reason is that gnssro_bufr2ioda produces ioda v1 and gnssro_bufr2ioda2 produces ioda v2, then I think it would be better to organize it as one executable that can write both formats. For that matter, it seems you could just update gnssro_bufr2ioda with the contents of gnssro_bufr2ioda2 since there is no need to continue supporting idoa v1 (at least that I'm aware of).

Reply:
Agree. Don't see a need for ioda v1. Could just upgrade this to v2 and use the original name.

Reply:
yes. the ioda1 converter was removed. Now the gnssro_bufr2ioda2.f90  is gnssro_bufr2ioda.f90

Reply:
@hailingz thanks for making this change. Now that gnssro_bufr2ioda.f90 has been updated with gnssro_bufr2ioda2.f90, please remove gnssro_bufr2ioda2.f90 and this ecbuild_add_executable command (since these are no longer necessary). Thanks!